static/app/components/hotkeys.tsx

```diff
@@ -0,0 +1,113 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {getKeyCode} from 'sentry/utils/getKeyCode';
+
+const macModifiers = {
+  18: '⌥',
+  17: '⌃',
+  91: '⌘',
+};
+
+const normalModifiers = {
+  18: 'ALT',
+  17: 'CTRL',
+  91: 'CMD',
+};
+
+const genericGlyphs = {
+  16: '⇧',
+  8: '⌫',
+  37: '←',
+  38: '↑',
+  39: '→',
+  40: '↓',
+  9999: '+',
+};
+
+const keyToDisplay = (
+  key: string,
+  platform?: 'macos' | string
+): [React.ReactNode, boolean] => {
+  let keyStr = key.toUpperCase();
+  let hasMissing = false;
+
+  const keyCode = getKeyCode(key);
+  if (keyCode) {
+    const isMac = platform
+      ? platform === 'macos'
+      : window?.navigator?.platform?.toLowerCase().startsWith('mac') ?? false;
+
+    const modifierMap = isMac ? macModifiers : normalModifiers;
+    const keyStr = modifierMap[keyCode] ?? key.toUpperCase();
+    
+    if (keyStr === 'CMD') {
+      hasMissing = true;
+    }
+
+    const glyph = genericGlyphs[keyCode];
+
+    if (glyph) {
+      keyStr = glyph;
+    }
+  }
+
+  // eslint-disable-next-line react/jsx-key
+  return [<Key>{keyStr}</Key>, hasMissing];
+};
+
+const Hotkeys = ({
+  value,
+  platform,
+}: {
+  /**
+   * Pass key combinations in with + as the separator.
+   * For example: command+option+x
+   *
+   * Use comma for fallback key combos when the first one contains a key that does not exist on that os (non-mac):
+   * command+option+x,ctrl+shift+x
+   * (does not have to be the same combo)
+   *
+   * Escape the + key with a slash |+
+   */
+  value: string;
+  platform?: 'macos' | string;
+}) => {
+  // Split by commas and then split by +, but allow escaped /+
+  const byPlatform = value.split(',').map(o => o.trim().split(/(?<!\\)\+/g));
```

Comment:
I wouldn't think of the list of key-combos as _specifically_ for providing different key-combos for different OS's (though it does allow this).

I think this is really just for allowing multiple shortcuts for one action.

In the scenario where they don't provide a mac AND windows variant, whatever OS just won't show. hotkey.

Reply:
For this component's case because it pertains to displaying the key combo rather than accepting it, I think multiple key bindings would probably make more sense to just have separate versions of this component rendered and this fallback mechanism is only to avoid displaying keys that won't work on the current platform. I like the idea of not rendering at all if they keys aren't present though. Let's do that.

Reply:
The reason I would think of it similar to the `useHotkeysHook` in terms of API for how it acepts the key list, is because then you can use the same value passed to the useHotkeysHook for this as well :)

Especially because, you can imagine a world where we can also associate a `description` to the `useHotkeysHook`, and have it store the hotkey and description into list in a react context, and then we could have a modal to show all active hotkeys and their descriptions. and ofc, we can use this component to show the hotkeys :)